site/docs/guide/getting-started.md

```diff
@@ -150,6 +150,116 @@ in your terminal before you execute `node bot.js`.
 This makes it easier to debug your bot.
 :::
 
+## Getting Started on Bun
+
+> This guide assumes that you have [Bun](https://bun.sh) installed.
+> If you don't know what these things are, check out our [Introduction](./introduction)!
+
+Create a new project and install the `grammy` package.
+Do this by opening a terminal and typing:
+
+```sh
+# Create a new directory and change into it.
+mkdir my-bot
+cd my-bot
+```
+
+Next step you should initialise your project.
+
+```sh
+# Run bun init to scaffold a new project.
```

Comment:
```suggestion
# Run bun init to scaffold a new project.
tsc —init
```

Reply:
Nice. Do we also need a `bun add -D typescript` somewhere in there, so that people have `tsc` ready? We might also want to use `bunx`.

Reply:
We can't init typescript before `bun init`.
`bun init` creates `package.json` and there could possibly be problems with that.

Reply:
We can just use bunx for that btw.
Like

```markdown
# Use bun to init Typescript config
bunx tsc --init
# Run bun init to scaffold a new project.
bun init
```

Reply:
Right, so adding TypeScript would be done after these two commands? That makes sense

Reply:
Wait, `bun init`  already includes a `tsconfig.json`, as explained in https://bun.sh/docs/typescript. I don't think this step is necessary, no?

Reply:
> Wait, `bun init` already includes a `tsconfig.json`, as explained in https://bun.sh/docs/typescript. I don't think this step is necessary, no?

See the earlier comments about keeping tsconfig consistent.